### PR TITLE
Bump micronaut-openapi to 2.1.0

### PR DESCRIPTION
**Pull request by micronaut-build**

### CODE REVIEW

**Comment by ilopmar** [comment]
Supersedes #4026

**Review by ilopmar — APPROVED** [review]

**Comment by graemerocher** [comment]
Also needs updating https://github.com/micronaut-projects/micronaut-gradle-plugin/blob/master/src/main/java/io/micronaut/gradle/MicronautLibraryPlugin.java#L37

**Comment by ilopmar** [comment]
The gradle plugin wasn't on my radar yet. I need to take a closer look at it

I've submitted this PR https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/18 although we can't merge it until we have a Micronaut 2.1 non-snapshot version that includes the group in the bom.

In any case I think we can merge because the plugin will use the old group until we merge the mentioned PR.

**Pull request merged**
